src/AddOns/BassFx/BassFx.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
     <AssemblyName>ppy.ManagedBass.Fx</AssemblyName>
-    <Version>3.0.0</Version>
+    <Version>2023.0.0</Version>
```

Comment:
Was this intentionally changed?

Reply:
Internal testing, I think I forgot to revert that. I'll make sure to do that